lib/modules/platform/github/graphql.ts

```diff
@@ -85,3 +85,36 @@ mutation EnablePullRequestAutoMerge(
   }
 }
 `;
+
+export const getIssuesByIssueTypeQuery = `
```

Comment:
Not sure we want this in this file, as it's not part of the application itself. Please move to under tools/docs

Reply:
Could we use the `gh` command, or some more native github library, instead of all this graphql code?

Reply:
We can use the `gh` command. Will need to parse the string o/p for required info.